diagnose_agent_banks.py

```
@@ -0,0 +1,134 @@
+
+# Test services
+print("\n🧪 Service Health Checks:")
+
+# Test CUA server
+try:
+    response = requests.get("http://localhost:5002/health", timeout=2)
+    if response.status_code == 200:
+        print("   ✅ CUA Execution Server: HEALTHY")
+    else:
+        print("   ❌ CUA Execution Server: UNHEALTHY")
+except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)